### DISCUSSION TITLE
[QUESTION]: Unable to update pollTimeoutSeconds and timeoutPolicy

### DISCUSSION THREAD

**Discussion by dcore94** [Netflix/conductor#2978] [Q&A]
Both from UI and REST API I cannot alter timeoutPolicy and pollTimeoutSeconds on tasks (in 3.7.x). Workflow is update and version increased and also some other fields (like tasknames) are updated. But those timeout related fields no.
Any hints?

**Comment by apanicker-nflx** [comment]
I was able to update the pollTimeoutSeconds and timeoutPolicy from the API however, the changes made on the UI did not take effect. We will look into fixing this soon.

**Reply by dcore94** [reply]
Tried today and no. I'm not even able to change it through REST API. The information appears to be lost (pollTimeoutSeconds) or overwritten (timeoutPolicy is always TIME_OUT_WF).

**Reply by jxu-nflx** [reply]
Hello @dcore94 can you provide details of what is the call you made via REST API? Also, provide the full json body or screen shots for your workflow execution would be helpful as well.
